panther_driver/src/driver_node.py

```diff
@@ -19,6 +19,9 @@
 import yaml
```

Comment:
Please sort imports as mentioned in the style guide doc and removed those ones which are unused.

Reply:
Done ✓

Reply:
Please sort your imports alphabetically within specific groups

Reply:
Imports now should be sorted correctly